src/main.rs

```diff
@@ -8,14 +8,16 @@ use axum::{
 };
 use axum::{routing::get, Router};
 use axum_extra::extract::cookie::Key;
-use hot_or_not_web_leptos_ssr::state::canisters::Canisters;
 use hot_or_not_web_leptos_ssr::{app::App, state::server::AppState};
 use hot_or_not_web_leptos_ssr::{
     auth::server_impl::store::KVStoreImpl, fallback::file_and_error_handler,
 };
+use hot_or_not_web_leptos_ssr::{consts::OFF_CHAIN_AGENT_GRPC_URL, state::canisters::Canisters};
 use leptos::{get_configuration, logging::log, provide_context};
 use leptos_axum::handle_server_fns_with_context;
 use leptos_axum::{generate_route_list, LeptosRoutes};
+#[cfg(feature = "ssr")]
+use tonic::transport::Channel;
```

Comment:
I usually put these import inside the scope or use absolute imports so we don't add an extra `#[cfg]` block

See https://github.com/go-bazzinga/hot-or-not-web-leptos-ssr/